src/main/scala/is/hail/utils/IntIterator.scala

```diff
@@ -1,14 +1,24 @@
 package is.hail.utils
 
-abstract class IntIterator extends Iterator[Int] {
+import scala.collection.mutable
+
+abstract class IntIterator {
   def nextInt(): Int
-  override def next(): Int = nextInt()
+
+  def hasNext: Boolean
+
+  def toArray: Array[Int] = {
+    val b = new mutable.ArrayBuilder.ofInt
```

Comment:
I've now searched and found a few Int / Double builders in other classes and changed them too.